modules/snowflake-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/snowflake/Snowflake.scala

```diff
@@ -209,15 +205,16 @@ object Snowflake {
               case Statement.ManifestAdd(message) =>
                 val tableName = Fragment.const(qualify(Manifest.Name))
                 val types = Fragment.const0(s"parse_json('${message.types.asJson.noSpaces}')")
+                val jobStarted: String = message.timestamps.jobStarted.toString
```

Comment:
Hey @spenes I found an alternative way to fix this problem, and I wonder if you prefer it.

In [this section](https://github.com/snowplow/snowplow-rdb-loader/blob/4.2.2/modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/config/StorageTarget.scala#L162-L166) add:

```
props.put("timezone", "UTC")
```

And then we can continue to use `Timestamp` in the prepared statement.  I slightly prefer passing a Timestamp to the prepared statement, because it's slightly more explicit about how we expect doobie and the warehouse to handle the value.

Reply:
I don't mind but I am not sure if it has other advantage then slightly more type safety.  In the end, we don't want any timezone change in the warehouse side. Wouldn't it better to load it as string to make sure that is the case ? 

Also, we have same problem with Databricks and Redshift Loader too. We should look if they have also equivalent of this property.

Reply:
No, it does not have any other advantage other than slightly more type safety.

Either method is OK for me; the most important thing is it works.  I'm just slightly annoyed that we have to cast something to a string when it isn't a string!  So far both methods feel slightly fragile and that's making me uncomfortable; like we could easily break this in a future release without noticing.  But then that's why we're currently investing in more integration tests.